proto/alarm/v1/stream.proto

```diff
@@ -45,28 +45,47 @@ message AlarmLogEntry {
 
     string chart = 3;
     string name = 4;
-    uint64 batch_id = 5;
-    uint64 sequence_id = 6;
-    uint64 when = 7;
+    string family = 5;
+    uint64 batch_id = 6;
+    uint64 sequence_id = 7;
+    uint64 when = 8;
 
-    string config_hash = 8;
+    string config_hash = 9;
 
-    AlarmStatus status = 9;
-    AlarmStatus old_status = 10;
-    uint64 delay_up_to_timestamp = 11;
+    uint32 utc_offset = 10;
+    string timezone = 11;
+
+    // Paths that can be custom for the same alarm, but depend on installation path for each user. Should be here or in config ?
+    string exec_path = 12; 
+    string conf_source = 13;
+    string command = 14;
```

Comment:
I believe config payload is the appropriate place. Only if a user changes the config files these values are going to change right?

Reply:
Actually @lvrach we noticed that this contains the absolute path of the netdata installation which might be different for each user.

Reply:
Hmm, not really, since these will contain the full path of the installation, so it will contain different paths for the same configuration file depending where the user installed netdata...

Reply:
So, we'll just leave them here...